### PR TITLE
Add option to skip attempt to convert to class and go straight to cre…

### PR DESCRIPTION
**Pull request by acdlite**
…ate-react-class fallback

### CODE REVIEW

File: transforms/__testfixtures__/class/class-no-conversion.output.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var React = require('React');
+
+var createReactClass = require('create-react-class');
```
Comment (bvaughn) [inline transforms/__testfixtures__/class/class-no-conversion.output.js]: This recast issue is annoying, huh :grin:

File: transforms/class.js [hunk 2]
```diff
@@ -1193,20 +1204,14 @@ module.exports = (file, api, options) => {
...
           fallbackToCreateClassModule(childPath);
         }
       });
-      return result;
-    };
-
-    const {
-      didTransform,
-      didFallback,
-    } = apply(ReactUtils.findAllReactCreateClassCalls(root));
```
Comment (bvaughn) [inline transforms/class.js]: FWIW this was the only use of `findAllReactCreateClassCalls` util method in the repo

File: transforms/class.js [hunk 3]
```diff
@@ -1178,12 +1180,21 @@ module.exports = (file, api, options) => {
...
-      const result = {};
+    let didTransform = false;
+    let didFallback = false;
+
+    const path = ReactUtils.findAllReactCreateClassCalls(root);
+    if (NO_CONVERSION) {
+      path.forEach(childPath => {
+        fallbackToCreateClassModule(childPath);
+      });
+      didTransform = false;
```
Comment (bvaughn) [inline transforms/class.js]: nit this isn't necessary

File: transforms/__testfixtures__/class/class-no-conversion.output.js (hunk 1 above)
Comment (acdlite) [inline transforms/__testfixtures__/class/class-no-conversion.output.js]: Meh, oh well :D

File: transforms/class.js (hunk 2 above)
Comment (acdlite) [inline transforms/class.js]: I moved it up to line 1186

**Pull request merged**